packages/clay-autocomplete/src/ClayAutocomplete.js

```diff
@@ -281,7 +305,13 @@ ClayAutocomplete.STATE = {
 	 * @memberof ClayAutocomplete
 	 * @type {?bool}
 	 */
-	enableAutocomplete: Config.bool().value(true),
+	enableAutocomplete: Config.validator(value => {
+		if (value) {
```

Comment:
hey @carloslancha, I know we're issuing this warning not to cause a change break. Maybe we should keep this API working too so we do not break the applications of who it is using... makes sense?

Reply:
Since to use your own filtering strategy you must be listening the event and passing `filtereItems` again to the components... I think nobody would be just passing `enableAutocomplete` without listening, so this is probably useless and will not break a thing.

Reply:
Oh yeah, good point, really this will not break. Thanks @carloslancha.